### PR TITLE
exclude-seqs tutorial

### PR DESCRIPTION
**Pull request by nbokulich**

### CODE REVIEW

File: source/tutorials/quality-control.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.. command-block::
+   :no-exec:
+
+   mkdir quality-control-tutorial
+   cd quality-control-tutorial
+
+Excluding sequences by alignment
+--------------------------------
+
+The `exclude-seqs` method aligns a series of query sequences contained in a ``FeatureData[Sequences]`` file against a series of reference sequences. This method employs a number of different alignment criteria (BLAST evalue, percent identity to top reference sequence, and percent of query that aligns to top reference sequence) to decide whether that sequence "hits" the reference sequences, and outputs separate files of sequences that hit/do not hit the reference sequences. This method can be used for a variety of applications, including removing known contaminant sequences, excluding host sequences (e.g., human DNA), or removing non-target sequences (e.g., non-bacterial) from your data.
```
Comment (jairideout) [inline source/tutorials/quality-control.rst]: Use double backticks to have `exclude-seqs` render correctly (it's currently italicized).

Comment (jairideout) [reply source/tutorials/quality-control.rst]: Change `FeatureData[Sequences]` to `FeatureData[Sequence]` (we don't have the plural form defined yet).

Comment (jairideout) [reply source/tutorials/quality-control.rst]: You mention "series of query sequences" and "series of reference sequences". The word "series" implies order -- does the order matter here? If not, perhaps "set of query sequences" and "set of reference sequences" would be clearer.

File: source/tutorials/quality-control.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+.. command-block::
+
+   qiime quality-control exclude-seqs \
+     --i-feature-sequences query-seqs.qza \
+     --i-reference-sequences reference-seqs.qza \
+     --p-method blast \
+     --p-perc-identity 0.97 \
+     --p-perc-query-aligned 0.97 \
+     --o-sequence-hits hits \
```
Comment (jairideout) [inline source/tutorials/quality-control.rst]: Can you include the `.qza` file extension on `hits` and `misses`?

File: source/tutorials/quality-control.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+.. download::
+   :url: https://data.qiime2.org/2017.10/tutorials/quality-control/reference-seqs.qza
+   :saveas: reference-seqs.qza
+
+Next, we will separate a small set of query sequences into those that hit/do not hit a set of reference sequences.
+
+.. command-block::
+
+   qiime quality-control exclude-seqs \
+     --i-feature-sequences query-seqs.qza \
```
Comment (jairideout) [inline source/tutorials/quality-control.rst]: Should this option be called `query-sequences` instead? Both query and reference inputs are "feature sequences" (i.e. `FeatureData[Sequence]`, and the tutorial exclusively refers to this input type as "query sequences".

File: source/tutorials/quality-control.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+     --i-reference-sequences reference-seqs.qza \
+     --p-method blast \
+     --p-perc-identity 0.97 \
+     --p-perc-query-aligned 0.97 \
+     --o-sequence-hits hits \
+     --o-sequence-misses misses
+
+This method currently supports ``BLAST``, ``vsearch``, and ``blastn-short`` as alignment methods. Note that ``blastn-short`` should be used if the query sequences include very short sequences (< 30 nt).
```
Comment (jairideout) [inline source/tutorials/quality-control.rst]: I think `BLAST` should be lowercase to match the casing required by `--p-method` in the above command.

File: source/tutorials/quality-control.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (jairideout) [inline source/tutorials/quality-control.rst]: Would you consider adding a short section on how to filter the feature table, and link to the filtering tutorial for more details?

File: source/tutorials/quality-control.rst (hunk 1 above)
Comment (nbokulich) [reply source/tutorials/quality-control.rst]: 👍

File: source/tutorials/quality-control.rst (hunk 2 above)
Comment (nbokulich) [reply source/tutorials/quality-control.rst]: 👍

File: source/tutorials/quality-control.rst (hunk 4 above)
Comment (nbokulich) [reply source/tutorials/quality-control.rst]: 👍

File: source/tutorials/quality-control.rst (hunk 5 above)
Comment (nbokulich) [reply source/tutorials/quality-control.rst]: 👍

**Comment by nbokulich** [comment]
Will not work until this PR is merged: https://github.com/qiime2/q2-quality-control/pull/18

**Comment by nbokulich** [comment]
thanks @jairideout ! I have pushed my latest changes — accepted all suggestions.

File: source/tutorials/quality-control.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+     --o-sequence-misses misses.qza
+
+This method currently supports ``blast``, ``vsearch``, and ``blastn-short`` as alignment methods. Note that ``blastn-short`` should be used if the query sequences include very short sequences (< 30 nt).
+
+Now that you have split your sequences into groups of sequences that hit/miss the reference sequences, you will most likely want to filter your feature table to remove hits or misses prior to further analysis. Filtering features from feature tables is fully covered in the :doc:`filtering tutorial <filtering>`, but here we will demonstrate an example of filtering a feature table using sequence data. In some cases, you may want to remove the `misses` from your feature table, e.g., if you are trying to select sequences that align to bacterial sequences (or a more specific clade). In other cases, you may want to remove the `hits` from your feature table, e.g., if you are trying to filter out contaminants or sequences that align to host DNA. Here we will filter out `hits` to demonstrate how to filter sequences from a filter table; one can replace ``hits.qza`` with ``misses.qza`` in the following command to filter out misses instead.
```
Comment (jairideout) [inline source/tutorials/quality-control.rst]: This example is actually filtering the table to *include* only the hits. Supplying `misses.qza` instead will accomplish what the text describes, or you could use `--p-exclude-ids` with `hits.qza`. Can you update the text and/or command (and output file names) to match whatever inverted filtering strategy you think makes sense?

**Comment by nbokulich** [comment]
thanks for catching @jairideout ! Let me know if the following changes make the most sense.

**Pull request merged**